generators/csharp/sdk/src/endpoint/EndpointGenerator.ts

```diff
@@ -32,7 +32,9 @@ export class EndpointGenerator extends AbstractEndpointGenerator {
         rawClient: RawClient;
         grpcClientInfo: GrpcClientInfo | undefined;
     }): csharp.Method {
-        if (grpcClientInfo != null) {
+        // If the service is a grpc service, grpcClientInfo will not be null or undefined,
+        // so any endpoint will be generated as a grpc endpoint, unless the transport is overriden by setting type to http
```

Comment:
nit: I think we can remove this comment - the control flow speaks for itself in my opinion. Thoughts?